### PR TITLE
Update arcade-validation steps

### PR DESCRIPTION
**Pull request by epananth**
### To double check:

* [ ] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/core-eng/tree/master/Documentation/Validation

### CODE REVIEW

**Comment by epananth** [comment]
@riarenas @MattGal please take a look at this doc. 

I updated this when I saw some discrepancies during branching.

File: Documentation/Policy/ArcadeServicing.md [hunk 1]
```diff
@@ -84,6 +84,18 @@ which Arcade can be branched for major release 'N' of .NET Core.
...
 7. Modify the new release branch  `release/<N>` of arcade-validation to promote builds
    to `.NET <N> Eng`. ([example](https://github.com/dotnet/arcade-validation/pull/1857/files))
+   In the update-channel.ps1 remove call to 
+   - Get-AzDO-Build (https://github.com/dotnet/arcade-validation/blob/71d3397e87e13ae2e9fd9d9af9725cfb53942c55/eng/validation/update-channel.ps1#L22)
+   - Get-AzDOHeaders (https://github.com/dotnet/arcade-validation/blob/71d3397e87e13ae2e9fd9d9af9725cfb53942c55/eng/validation/update-channel.ps1#L30)
+   - Get-LatestBuildResult (https://github.com/dotnet/arcade-validation/blob/71d3397e87e13ae2e9fd9d9af9725cfb53942c55/eng/validation/update-channel.ps1#L37)
+   - Remove the reference to bellweather repos (https://github.com/dotnet/arcade-validation/blob/main/eng/validation/update-channel.ps1#L54-#L76) and (https://github.com/dotnet/arcade-validation/blob/71d3397e87e13ae2e9fd9d9af9725cfb53942c55/eng/validation/update-channel.ps1#L85)
+      In azure-pipeline.yml
```
Comment (MattGal) [inline Documentation/Policy/ArcadeServicing.md:92]: file name is azure-pipelines.yml

**Review by MattGal — APPROVED** [review]
Seems good to me, I wonder if it'd be better to describe the location instead of locking in to a commit in the yaml that could get really old

**Review by riarenas — APPROVED** [review]

File: Documentation/Policy/ArcadeServicing.md (hunk 1 above)
Comment (epananth) [reply Documentation/Policy/ArcadeServicing.md]: Updated this

**Pull request merged**